apps/web/src/features/athletes/components/athlete.detail.skeleton.tsx

```
@@ -0,0 +1,75 @@
+								<div className="h-6 w-16 rounded bg-muted" />
+								<div className="h-6 w-24 rounded bg-muted" />
+							</div>
+						</div>
+					</div>
+					<div className="h-10 w-32 rounded bg-muted" />
+				</div>
+
+				{/* Cards skeleton */}
+				<div className="grid gap-6 md:grid-cols-2 lg:grid-cols-3">
+					{[...Array(6)].map((_, i) => (
+						<div key={i} className="rounded-lg border p-6">
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)